lib/active_fedora/relation/finder_methods.rb

```diff
@@ -189,17 +189,16 @@ def find_one(id, cast = nil)
     protected
 
       def load_from_fedora(id, cast)
-        raise ActiveFedora::ObjectNotFoundError if id.empty?
+        if id.empty?
```

Comment:
We should favor the if/unless modifer style per https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier